datastore/elastic_search/query.py

```diff
@@ -98,6 +99,57 @@ def ngrams_query(connection, index_name, doc_type, entity_name, ngrams_list, fuz
     return ngram_results
 
 
+def user_says_query(connection, index_name, doc_type, entity_name, user_typed_sentence, fuzziness_threshold,
+                    search_language_script=None, **kwargs):
+    """
+    Performs compound elasticsearch boolean search query with highlights for the given user sentence . The query
```

Comment:
Remove terms such as 'user', 'sentence', etc and use generic term such as 'text',etc.